packages/pluggableWidgets/rich-text-web/src/utils/ckeditorPresets.ts

```diff
@@ -0,0 +1,156 @@
+        items: ["Form", "Checkbox", "Radio", "TextField", "Textarea", "Select", "Button", "ImageButton", "HiddenField"]
+    },
+    {
+        name: "basicstyles",
+        items: [
+            "Bold",
+            "Italic",
+            "Underline",
+            "Strike",
+            "Subscript",
+            "Superscript",
+            "-",
+            "CopyFormatting",
+            "RemoveFormat"
+        ]
+    },
+    {
+        name: "paragraph",
+        items: [
+            "NumberedList",
+            "BulletedList",
+            "-",
+            "Outdent",
+            "Indent",
+            "-",
+            "Blockquote",
+            "CreateDiv",
+            "-",
+            "JustifyLeft",
+            "JustifyCenter",
+            "JustifyRight",
+            "JustifyBlock",
+            "-",
+            "BidiLtr",
+            "BidiRtl",
+            "Language"
+        ]
+    },
+    { name: "links", items: ["Link", "Unlink", "Anchor"] },
+    { name: "insert", items: ["Image", "Table", "HorizontalRule", "Smiley", "SpecialChar", "PageBreak", "Iframe"] },
+    { name: "styles", items: ["Styles", "Format", "Font", "FontSize"] },
+    { name: "colors", items: ["TextColor", "BGColor"] },
+    { name: "tools", items: ["Maximize", "ShowBlocks"] },
+    { name: "about", items: ["About"] }
+];
+
+function SET_PRESET(type: "basic" | "standard" | "full"): CKEditorConfig {
+    const config: CKEditorConfig = {};
+    let toolbarGroup: Array<ToolbarGroup | string> = [];
+    switch (type) {
+        case "basic":
+            config.toolbarGroups = [...TOOLBAR_GROUP];
+            config.removeButtons =
+                "Source,Save,Templates,NewPage,ExportPdf,Preview,Print,Cut,Redo,Copy,Undo,Paste,PasteText,PasteFromWord,Find,Replace,SelectAll,Scayt,Form,Checkbox,Radio,TextField,Textarea,Select,Button,ImageButton,HiddenField,Strike,Underline,Subscript,Superscript,CopyFormatting,RemoveFormat,CreateDiv,Blockquote,JustifyLeft,BidiLtr,Image,Table,BidiRtl,JustifyCenter,JustifyRight,Language,JustifyBlock,HorizontalRule,SpecialChar,Smiley,PageBreak,Iframe,Styles,TextColor,BGColor,ShowBlocks,Maximize,Format,Font,FontSize,Anchor";
+            break;
+        case "standard":
+            toolbarGroup = [...TOOLBAR_GROUP];
+            toolbarGroup.splice(4, 0, "/");
+            config.toolbarGroups = toolbarGroup;
+            config.removeButtons =
+                "Save,Templates,NewPage,ExportPdf,Preview,Print,Find,Replace,SelectAll,Form,Checkbox,Radio,TextField,Textarea,Select,Button,ImageButton,HiddenField,Underline,Subscript,Superscript,CopyFormatting,CreateDiv,JustifyLeft,BidiLtr,BidiRtl,JustifyCenter,JustifyRight,Language,JustifyBlock,Smiley,PageBreak,Iframe,TextColor,BGColor,Font,FontSize,ShowBlocks";
+            break;
+        case "full":
+            toolbarGroup = [...TOOLBAR_GROUP];
+            toolbarGroup.splice(3, 0, "/");
+            toolbarGroup.splice(5, 0, "/");
+            toolbarGroup.splice(6, 0, "/");
```

Comment:
https://github.com/mendix/widgets-resources/pull/1125#discussion_r769480338
Later, in case when we receive feedback from customer that they want to define their own  break lines, we can change index's with their inputs

Reply:
Is there any meaning behind these numbers/indices? Right now, they are quite arbitrary and it will be very difficult to understand them in the future.

Reply:
Have you checked the comment above?